azure-pipelines.sds.pr-release.yml

```diff
@@ -48,10 +49,59 @@ stages:
             imageTags:
               - '${{ variables.prTagName }}'
 
-  - stage: Install
+#####################################################
+# Run Entity Framework Dev. #########################
+  - stage: EF_Dev
     dependsOn: Dock
+    displayName: Run EF Dev
+    variables:
+      - group: vh-sql-credentials
+      - template: variables/dev.yaml
+      - template: variables/shared.yaml
+        parameters:
+          env: ${{ variables.env }}
+          cluster: $(dev_cluster)
+    jobs:
+      - job: Run_Entity_Framework_Dev
+        pool:
+         vmImage: 'windows-latest'
+        displayName: Run EF Dev
+        steps:
+          - template: templates/Database/Firewall/addSqlFirewallRule.yaml@azTemplates
+            parameters:
+              sqlServerResourceGroup: ${{ variables.vhResourceGroup }}
+              sqlServerName: ${{ variables.vhSQLServerName }}
+              azureSubscription: ${{ variables.subscriptionName }}
+
+          - template: templates/Database/EntityFramework/generate-script.yaml@azTemplates
+            parameters:
+              outputPath: $(Build.StagingDirectory)
+              contextName: BookingsDbContext
```

Comment:
Can context name and project name be put in variables so we don't repeat?
The variables can the be used in both dev and demo plus the release pipeline

Reply:
amended.